boxoffice/assets/js/templates/org.html.js

```diff
@@ -21,13 +21,13 @@ export const orgTemplate = `
             <p class="heading-title">{{ title }}</p>
           {{/title}}
           <div class="heading-edit">
-            <button class="edit-btn" on-click="showEditICForm(event)"><i class="fa fa-edit"></i>{{#loadingEditForm}}<i class="fa fa-spinner fa-spin">{{/}}</button>
+            <button class="edit-btn" on-click="showIcForm(event, 'edit')"><i class="fa fa-edit"></i>{{#loadingEditForm}}<i class="fa fa-spinner fa-spin">{{/}}</button>
           </div>
         </div>
         <div class="content">
           {{#if showEditForm}}
             <div class="content-box clearfix" intro='fly:{"x":20,"y":"0"}'>
-              <EditICForm formTemplate="{{ formTemplate }}" ic="{{ item_collection }}" icId="{{ id }}"></EditICForm>
+              <ICForm formTemplate="{{ formTemplate }}" ic="{{ item_collection }}" icId="{{ id }}"></ICForm>
```

Comment:
Can we abstract this further and make it `Form`? That way we can reuse this across models?